docs/blog/newsletter-march-2020.rst

```diff
@@ -0,0 +1,140 @@
+.. post:: March 05, 2020
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – March 2020
+######################################
+
+Hello everyone! March is here (already?!), and so is a fresh new edition of the WTD newsletter.
+
+Firstly, announcing some news I know some of you have been waiting for. We're delighted that the `results of last year's salary survey </surveys/salary-survey/2019/>`__ are now live! Openness is a core value of this community, so it's great to be able to present the anonymized data for your perusing.
+
+On the conference side, it's exciting to see the `awesome roster of speakers for Portland </conf/portland/2020/news/announcing-speakers/#full-speaker-line-up>`__ - some seriously great talks to look forward to.
+
+We do also want to address the situation with COVID-19. Currently, *we have no plans to cancel* the Portland conference, but are planning to take additional precautions during the on-site events. We'll be watching the situation closely over the next few weeks, and taking advice from the Oregon health authorities and United States CDC as it comes. We will assess the situation again next week and provide updates as the situation changes. 
```

Comment:
It might be valuable to say where we're going to provide those updates. Like should I look in #announcements and #wtd-conferences? Expect an email? What channels should I be watching, you know.

Reply:
Good idea. #1190 contains link to a the blog archive of covid-19 tag, which would do the trick.

Reply:
We now have an official announcement to link to: https://www.writethedocs.org/conf/portland/2020/news/covid-19-update-01/

Reply:
Added link and reference to what we'll update.